publish/publish-to-sonatype.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-./publish/publish-to-sonatype-212.sh
```

Comment:
2.13 not published

Reply:
@echeipesh hm do you think we want to publish 3.6.0 on MavenCentral with the Spark 3.2.0-SNAPSHOT dep?

Reply:
I think we should publish 3.6.0-SNAPSHOT with spark 3.2.0-SNAPSHOT dep, which would happen. Our release process is not setup for a split release though ... maybe the problem will solve itself with release of spark 3.2.0 before we actually have to cut 3.6.0 release. I don't think there is anything forcing our release to happen soon?

Reply:
Yea, that is why 2.13 is not published, these scripts are to publish release artifacts.